react/components/PageHeader/index.js

```diff
@@ -33,22 +33,34 @@ class PageHeader extends PureComponent {
           </div>
         )}
 
-        <div
-          className={`vtex-pageHeader__title c-on-base t-heading-2 flex flex-row
-          flex-wrap justify-between ${linkLabel ? 'mt0' : 'mt7'}`}>
-          <span>{this.props.title}</span>
-
-          {children && (
-            <div className="vtex-pageHeader__children">{children}</div>
-          )}
+        <div>
```

Comment:
This was added because of flex?

Reply:
It was, but became unnecessary. Removed.